packages/ethereum-contracts/contracts/agreements/ConstantFlowAgreementV1.sol

```diff
@@ -1055,8 +1064,8 @@ contract ConstantFlowAgreementV1 is
                         int256 userDamageAmount = AgreementLibrary.min(
                             // user will take the damage if the app is broke,
                             -availableBalance,
-                            // but user's damage is limited to the amount of app allowance it gives to the app
-                            AgreementLibrary.max(0, -appAllowanceDelta));
+                            // but user's damage is limited to the amount of app credit it gives to the app
+                            -appCreditDelta);
```

Comment:
⚠️ new logic. supposedly appCreditDelta is always positive

Reply:
added comments to explain why this should be the case